pandas/tests/io/parser/test_parse_dates.py

```diff
@@ -848,6 +846,34 @@ def test_parse_dates_arrow_engine(all_parsers):
     tm.assert_frame_equal(result, expected)
 
 
+def test_parse_dates_arrow_dtype_as_index(all_parsers):
+    # GH#57930
+    parser = all_parsers
+    data = """a,b
+2000-01-01 00:00:00,1
+2000-01-01 00:00:01,1"""
+
+    result = parser.read_csv(
+        StringIO(data),
+        parse_dates=["a"],
+        index_col="a",
+        dtype_backend="pyarrow",
+    )
+    expected = Series(
+        [1, 1],
+        name="b",
+        dtype="int64[pyarrow]",
+        index=Index(
+            [
+                Timestamp("2000-01-01 00:00:00"),
+                Timestamp("2000-01-01 00:00:01"),
+            ],
+            name="a",
+        ),
+    ).to_frame()
+    tm.assert_frame_equal(result, expected)
```

Comment:
```suggestion
    )
    tm.assert_series_equals(result, expected)
```

Reply:
I probably need to change the `parser.read_csv` argument then as that results in a dataframe.
Will that not cause a problem comparing a dataframe to a pd.Series?

Reply:
Ah sorry I had assumed since you were using `to_frame` you could just use `assert_series_equal`. What you had before was OK then but instead of using `to_frame` could you just use the `pd.DataFrame` constructor instead?

Reply:
Fair question to which I have no answer... 
I change the constructor.